app/views/users/registrations/new.html.erb

```diff
@@ -38,6 +38,13 @@
       <span><small><%= t 'users.registrations.new.team_name_help' %></small></span>
     </div>
 
+    <% if Rails.configuration.x.enable_recaptcha %>
```

Comment:
I would style this at least a bit in case of an error (e.g.):
```erb
<% if Rails.configuration.x.enable_recaptcha %>
      <div class="form-group <%= 'has-error' if flash[:recaptcha_error] %>">
        <%= recaptcha_tags %>
        <% if flash[:recaptcha_error] %>
          <span class="help-block">
            <%= flash[:recaptcha_error] %>
          </span>
        <% end %>
      </div>
    <% end %>
```
Or even better, since this will be used on multiple parts of the application, maybe even refactor it into a helper method (e.g. `recaptcha_input_tag`).

If you'd refactor it, I would make a new helper (e.g. `form_tag_helper.rb` or something like that).